test/util/base_api_mock.js

```diff
@@ -39,11 +39,20 @@ class BaseApiMock extends ApiMock {
       low: 9_000,
       volume: 3
     })
+    const baseTrade = new PublicTrade({ price: 5000, amount: 9001 })
+    const baseBook = new OrderBook([
+      [67536333723, 500, -2],
+      [67538314922, 600, -2],
+      [67535007022, 500, 2],
```

Comment:
both heads at `500`, wouldn't they have matched?

Reply:
should be `499` for the buy instead?

Reply:
Yeah, it can, but it seems that in this situation it does not affect